### ISSUE TITLE
ERROR for the site owner: Invalid stoken

### ISSUE THREAD

**Issue by fsaalmeidinha** [VividCortex/angular-recaptcha#147]
I've configured the reCaptcha according to your example:

`angular.module('myapp').config(reCaptcha);

```
reCaptcha.$inject = ['vcRecaptchaServiceProvider'];

function reCaptcha(vcRecaptchaServiceProvider) {

    vcRecaptchaServiceProvider.setSiteKey('Site key');
    vcRecaptchaServiceProvider.setTheme('light');
    vcRecaptchaServiceProvider.setStoken('Secret key');
    vcRecaptchaServiceProvider.setSize('normal');
    vcRecaptchaServiceProvider.setType('image');

}`
```

Testing the reCaptcha, i've got the following error: "ERRO para o proprietário do site: Invalid stoken".

I made a test commenting the line of code that configures the STOKEN:
vcRecaptchaServiceProvider.setStoken('Secret key');

And the reCaptcha has started to work
I thought it really weird working without STOKEN and presenting the same error with STOKEN..

Can you help me solving this problem??
Thanks!!

**Comment by TheSharpieOne** [comment]
The stoken provided via [the services provider `setStoken` call](https://github.com/VividCortex/angular-recaptcha/blob/master/src/service.js#L55-L57) only stores the variable [to be used](https://github.com/VividCortex/angular-recaptcha/blob/master/src/service.js#L142) in the case when it is not [provided](https://github.com/VividCortex/angular-recaptcha/blob/master/src/service.js#L138) on the [directive level](https://github.com/VividCortex/angular-recaptcha/blob/master/src/directive.js#L48).
Basically, angular-recaptcha only passes the value you provide to it to the reCaptcha instance. It seems as if the stoken you are using is invalid.
Perhaps you are mixing up the site key with the stoken or copied the stoken incorrectly?

**Comment by mtrias** [comment]
Closing for inactivity, please reopen if you are still having this issue.
